ExampleMarkdownDocument.md

```diff
@@ -0,0 +1,93 @@
+# Example Markdown Document
+
+Version: 1.0.1  
+13.01.2017  
```

Comment:
Is it enough if the date is in the version history table below and we could leave this date out?